_studio/shared/include/mfx_config.h

```diff
@@ -68,7 +68,7 @@
         #define MFX_ENABLE_H264_VIDEO_FEI_PAK
     #endif
     #if defined(MFX_ENABLE_MCTF)
-        //#define MXF_ENABLE_MCTF_IN_AVC
+        //#define MFX_ENABLE_MCTF_IN_AVC
```

Comment:
I am not sure why do you have 2 commits instead of 1. However, I think you need one more (separate!) commit to enable fixed MCTF path.

Reply:
Yes, I will discard this draft and do a clean commit at the end